src/components/IKUpload/props.ts

```diff
@@ -44,6 +68,7 @@ export type IKUploadProps = InferProps<typeof Props> & {
     onUploadStart?: (evt: React.ChangeEvent<HTMLInputElement>) => void;
     onUploadProgress?: (evt: ProgressEvent<XMLHttpRequestEventTarget>) => void;
     validateFile?: (file: File) => boolean;
```

Comment:
Should we update type definition for extensions and other related improvements?

Reply:
done